### PR TITLE
21 Add a product to the business

### PR DESCRIPTION
**Pull request by leonardk1**
#21 
As An Assigned Contact
I Want To Add A Product to the Business
So That I can Be able to Create Orders for that Product

##### Acceptance Criteria
- [x] GIVEN I Want To Add a New Product to the Business, THEN there should be a link to Add the New Product
- [x] GIVEN I Click the Link to Add the New Product, THEN There should be a form to Add the new Product details
- [x] GIVEN I have Added and Clicked the Submit Button THEN I should be given notification for success or failure and I should also be shown the newly added product details

### CODE REVIEW

**Comment by leonardk1** [comment]
@godfreym please review

File: spec/controllers/products_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe ProductsController, :type => :controller do
+  before do   
+    @business = create(:business)
+    sign_in create(:contact, business_id: @businessg)
```
Comment (godfreym) [inline spec/controllers/products_controller_spec.rb]: @leonardk1 did you run these tests? what is `@businessg` ? note the g

Comment (leonardk1) [inline spec/controllers/products_controller_spec.rb]: @godfreym  yes I did, I wonder why it Did not catch it

**Comment by godfreym** [comment]
@leonardk1 :+1:

**Comment by leonardk1** [comment]
@arafatm please review

**Comment by arafatm** [comment]
@leonardk1 :+1:

**Pull request merged**